OtripleS.Web.Api.Tests.Unit/Services/UserServiceTests/UserServiceTests.Logic.cs

```diff
@@ -0,0 +1,56 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using OtripleS.Web.Api.Models.Users;
+using Xunit;
+
+namespace OtripleS.Web.Api.Tests.Unit.Services.UserServiceTests
+{
+    public partial class UserServiceTests
+    {
+        [Fact]
+        public async Task ShouldCreateUserAsync()
+        {
+            // given
+            DateTimeOffset randomDateTime = GetRandomDateTime();
+            DateTimeOffset dateTime = randomDateTime;
+            User randomUser = CreateRandomUser(randomDateTime);
```

Comment:
@hassanhabib here i get the following error:
I can not make the test ShouldCreateUserAsync PASS. I got error:

System.TypeInitializationException : The type initializer for 'System.Nullable1[[System.DateTimeOffset, System.Private.CoreLib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]]' threw an exception. ---- System.Exception : The type [Nullable1] was not registered in the randomizer.
at Tynamix.ObjectFiller.Filler1.GetRandomValue(Type propertyType, FillerSetupItem setupItem) at Tynamix.ObjectFiller.Filler1.CreateAndFillObject(Type type, FillerSetupItem currentSetupItem, HashStack1 typeTracker) at Tynamix.ObjectFiller.Filler1.FillProperties(Object objectToFill, PropertyInfo[] properties, FillerSetupItem currentSetup, HashStack1 typeTracker) at Tynamix.ObjectFiller.Filler1.FillInternal(Object objectToFill, HashStack1 typeTracker) at Tynamix.ObjectFiller.Filler1.Fill(T instanceToFill)
at Tynamix.ObjectFiller.Filler`1.Create()
at OtripleS.Web.Api.Tests.Unit.Services.UserServiceTests.UserServiceTests.CreateRandomUser(DateTimeOffset dates) in /Users/eriadhami/Projects/OtripleS/OtripleS.Web.Api.Tests.Unit/Services/UserServiceTests/UserServiceTests.cs:line 40
at OtripleS.Web.Api.Tests.Unit.Services.UserServiceTests.UserServiceTests.ShouldCreateUserAsync() in /Users/eriadhami/Projects/OtripleS/OtripleS.Web.Api.Tests.Unit/Services/UserServiceTests/UserServiceTests.Logic.cs:line 23
--- End of stack trace from previous location where exception was thrown ---
----- Inner Stack Trace -----

Look like i have problems with my `User` model or with the `Filler`

Reply:
@hassanhabib looks like Objectfiller.NET has problems with filling a `IdentityUser`. Do i have to continue to fill the User manually for each property? Something like:

```
private User CreateRandomUser()
        {
            var user = new User
            {
                Id = new Guid(),
                UserName = new Filler<EmailAddresses>().Create().ToString(),
                Name = new Filler<RealNames>().Create().ToString(),
                FamilyName = new Filler<RealNames>().Create().ToString(),
                CreatedDate = DateTimeOffset.UtcNow,
                UpdatedDate = DateTimeOffset.UtcNow

            };

            return user;
        }
```